<launch>
    <param name="init_params" type="bool" value="true" />
    <rosparam param="crop_min_x">-600.919</rosparam>
    <rosparam param="crop_max_x">-141.307</rosparam>
    <rosparam param="crop_min_y">-342.415</rosparam>
    <rosparam param="crop_max_y">402.577</rosparam>
    <rosparam param="crop_min_z">-600.919</rosparam>
    <rosparam param="crop_max_z">24.6841</rosparam>
    <rosparam param="ax">1.34896</rosparam>
    <rosparam param="bx">810.618</rosparam>
    <rosparam param="ay">1.12753</rosparam>
    <rosparam param="by">386.083</rosparam>
    <rosparam param="az">1.59846</rosparam>
    <rosparam param="bz">960.543</rosparam>
    <node name="depth_detector" pkg="depth_perception" type="depth_perception_node" output="screen"/>
</launch>